pkg/virtctl/vm/add_volume_test.go

```diff
@@ -180,13 +193,14 @@ func createTestPVC() *k8sv1.PersistentVolumeClaim {
 	}
 }
 
-func verifyVolumeSource(volumeOptions *v1.AddVolumeOptions, useDv bool) {
+func verifyDVVolumeSource(volumeOptions *v1.AddVolumeOptions) {
 	Expect(volumeOptions.VolumeSource).ToNot(BeNil())
-	if useDv {
-		ExpectWithOffset(3, volumeOptions.VolumeSource.DataVolume).ToNot(BeNil())
-		ExpectWithOffset(3, volumeOptions.VolumeSource.PersistentVolumeClaim).To(BeNil())
-	} else {
-		ExpectWithOffset(3, volumeOptions.VolumeSource.DataVolume).To(BeNil())
-		ExpectWithOffset(3, volumeOptions.VolumeSource.PersistentVolumeClaim).ToNot(BeNil())
-	}
+	ExpectWithOffset(3, volumeOptions.VolumeSource.DataVolume).ToNot(BeNil())
+	ExpectWithOffset(3, volumeOptions.VolumeSource.PersistentVolumeClaim).To(BeNil())
+}
+
+func verifyPVCVolumeSource(volumeOptions *v1.AddVolumeOptions) {
+	Expect(volumeOptions.VolumeSource).ToNot(BeNil())
+	ExpectWithOffset(3, volumeOptions.VolumeSource.DataVolume).To(BeNil())
+	ExpectWithOffset(3, volumeOptions.VolumeSource.PersistentVolumeClaim).ToNot(BeNil())
 }
```

Comment:
WDYT about inlining these, as they are both used only a single time?

Reply:
```
		expectFunc(vmiName,
			volumeName,
			func(volumeOptions *v1.AddVolumeOptions) {
				Expect(volumeOptions.VolumeSource).ToNot(BeNil())
				ExpectWithOffset(3, volumeOptions.VolumeSource.DataVolume).ToNot(BeNil())
				ExpectWithOffset(3, volumeOptions.VolumeSource.PersistentVolumeClaim).To(BeNil())
			})
```
Imo it isn't good because it force the reader to go into details while reading high level